src/variables.py

```diff
@@ -100,6 +101,8 @@ def check_uri_required_variables():
         errors.append('CONSENSUS_CLIENT_URI')
     if '' in KEYS_API_URI:
         errors.append('KEYS_API_URI')
+    if not KUBO_HOST:
```

Comment:
I just thought that this is not the best idea, at least for local development.
Also, Oracles in infra will be instantly broken